blob/s3blob/s3blob.go

```diff
@@ -960,14 +968,19 @@ func (b *bucket) NewTypedWriter(ctx context.Context, key string, contentType str
 		}
 		if opts.BeforeWrite != nil {
 			asFunc := func(i interface{}) bool {
-				pu, ok := i.(**s3managerv2.Uploader)
-				if ok {
-					*pu = uploaderV2
+				// Note that since GCS does not expose AWS's
```

Comment:
What is "GCS"?

Reply:
Google Cloud Storage, I'd think. (I'm just passing by)

Reply:
That's what GCS means to me too, but this is the AWS S3 driver, not the GCS one.

Reply:
Interesting... yeah, I picked up "GCS" from [some code in the test file](https://github.com/google/go-cloud/blob/master/blob/s3blob/s3blob_test.go#L132), but I bet that code actually _did_ come from GCS.

Will fix!